src/utils.ts

```diff
@@ -49,12 +50,28 @@ export function formatErrors(
     loaderOptions: LoaderOptions,
     colors: Chalk,
     compiler: typeof typescript,
-    merge?: { file?: string; module?: WebpackModule }
+    merge: { file?: string; module?: WebpackModule },
+    context: string
 ): WebpackError[] {
 
     return diagnostics
         ? diagnostics
-            .filter(diagnostic => loaderOptions.ignoreDiagnostics.indexOf(diagnostic.code) === -1)
+            .filter(diagnostic => {
+                
```

Comment:
Could we lose this blank line please? Reminds me - must add prettier to ts-loader

Reply:
done